src/components/themed/ThemedScene.tsx

```diff
@@ -49,7 +49,8 @@ export function ThemedScene(props: Props) {
 
   const containerStyle = {
     flex: 1,
-    ...sidesToPadding(mapSides(fixSides(paddingRem, 0.5), theme.rem))
+    ...sidesToMargin(mapSides(fixSides(paddingRem, 0.5), theme.rem)),
+    marginBottom: 0
```

Comment:
I cannot approve this change so late in the release cycle because it affects every scene. Also, it breaks the API for the `ThemedScene` component, randomly ignoring the user's bottom margin.

Instead, pass `<ThemedScene paddingRem={[0.5, 0.5, 0]} ...` or such in the NewAccountTosScene, to manually tweak the margins you need.

Reply:
I think I had this before because I didn't fully fix the flex on the scene first. No longer needed.